sdk/communication/arm-communication/tsconfig.json

```diff
@@ -9,25 +9,11 @@
     "esModuleInterop": true,
     "allowSyntheticDefaultImports": true,
     "forceConsistentCasingInFileNames": true,
-    "lib": [
-      "es6",
-      "dom"
-    ],
+    "lib": ["es6", "dom"],
     "declaration": true,
     "outDir": "./dist-esm",
-    "importHelpers": true,
-    "paths": {
```

Comment:
Is this file autogenerated , if not why remove this?

Reply:
I didn't touch this file

Reply:
Can you undo the change here ? Thanks